src/gui/advancedsettings.cpp

```diff
@@ -609,6 +617,23 @@ void AdvancedSettings::loadAdvancedSettings()
     // Announce to all tiers
     m_checkBoxAnnounceAllTiers.setChecked(session->announceToAllTiers());
     addRow(ANNOUNCE_ALL_TIERS, tr("Always announce to all tiers"), &m_checkBoxAnnounceAllTiers);
+
+    // Peer Turnover
+    m_spinBoxPeerTurnover.setMinimum(0);
+    m_spinBoxPeerTurnover.setMaximum(100);
+    m_spinBoxPeerTurnover.setValue(session->peerTurnover());
+    m_spinBoxPeerTurnover.setSuffix(tr(" %"));
+    addRow(PEER_TURNOVER, tr("Percentage of peers to disconnect"), &m_spinBoxPeerTurnover);
```

Comment:
Also the description is a bit misleading/scary (all 3 of them), I think there is room for improvement.

Reply:
Trying to make the description short - but I will try to improve them.